public/v4/apps/invoiceninja.yml

```diff
@@ -0,0 +1,125 @@
+captainVersion: 4
+services:
+    $$cap_appname-php:
+        restart: always
+        networks: invoiceninja
+        volumes:
+            - $$cap_appname-data:/var/www/app
+        depends_on:
+            - $$cap_appname-db
+        environment:
+            APP_NAME: $$cap_appname
+            APP_URL: $$cap_appname.$$cap_root_domain
+            APP_KEY: $$cap_app_key
+            APP_DEBUG: $$cap_app_debug
+            MULTI_DB_ENABLED: 'false'
+            DB_CONNECTION: mysql
+            DB_HOST1: srv-captain--$$cap_appname-db
+            DB_PORT1: '3306'
+            DB_USERNAME1: $$cap_db_user
+            DB_PASSWORD1: $$cap_db_pass
+            DB_DATABASE1: $$cap_db_name
+            IS_DOCKER: 'true'
+            PHANTOMJS_PDF_GENERATION: $$cap_phantomjs_pdf_generation
+            PHANTOMJS_CLOUD_KEY: $$cap_phantomjs_cloud_key
+            BAK_PUBLIC_PATH: '/var/www/app/docker-backup-public'
+            NINJA_PROD: 'true'
+            ENV_LIST: 'NINJA_PROD,APP_NAME,APP_URL,APP_KEY,APP_DEBUG,MULTI_DB_ENABLED,DB_CONNECTION,DB_HOST1,DB_PORT1,DB_USERNAME1,DB_PASSWORD1,DB_DATABASE1,IS_DOCKER,PHANTOMJS_PDF_GENERATION,PHANTOMJS_CLOUD_KEY'
+        caproverExtra:
+            notExposeAsWebApp: 'true'
+            dockerfileLines:
+                - FROM invoiceninja/invoiceninja:$$cap_app_tag
+                - RUN curl -o create-env-file-from-sys-env.sh https://gist.githubusercontent.com/olragon/e2f3ececd70b8c513ce6f768b47b6e5f/raw/0ce9188a0cbf1a9e6a7be4128419282b30492f49/create-env-file-from-sys-env.sh
```

Comment:
It's okay for `create-env-file-from-sys-env.sh` script to be hosted on github gists as it's not mutable.

Alternatively, you can embed it here in the dockerfileLines, something like:
```
RUN echo "#!/bin/bash \n echo Hi" > script.sh && chmod +x script.sh
```

Your current method is better as it's more readable.